api/user.go

```diff
@@ -72,6 +72,8 @@ func InitUser() {
 	BaseRoutes.NeedUser.Handle("/audits", ApiUserRequired(getAudits)).Methods("GET")
 	BaseRoutes.NeedUser.Handle("/image", ApiUserRequiredTrustRequester(getProfileImage)).Methods("GET")
 
+	BaseRoutes.NeedTeam.Handle("/users/recently_active", ApiUserRequiredActivity(getRecentlyActiveUsers, false)).Methods("GET")
```

Comment:
Is this only used in the system console? I think it should be an admin only API and in admin.go then 3/5